js/samples/04.ai.b.messageExtensions.AI-ME/sample.env

```diff
@@ -0,0 +1,4 @@
+OPENAI_KEY=
```

Comment:
In the line above this one, add something like:
```suggestion
# This is an example file of how to set up required environment variables. You can copy this contents into a new .env file and add the below as appropriate`
OPENAI_KEY=
```

Reply:
Updating to 
"# This is an example file of how to set up environment variables. You can duplicate this file and add the appropriate keys."